Src/Support/Google.Apis.Auth/OAuth2/DefaultCredentialProvider.cs

```diff
@@ -204,21 +204,13 @@ internal GoogleCredential CreateDefaultCredentialFromJson(string json)
 
 
         /// <summary>Creates a default credential from JSON data.</summary>
-        private static GoogleCredential CreateDefaultCredentialFromParameters(JsonCredentialParameters credentialParameters)
-        {
-            switch (credentialParameters.Type)
+        internal GoogleCredential CreateDefaultCredentialFromParameters(JsonCredentialParameters credentialParameters) =>
```

Comment:
Add a nullity check? I suspect we previously "knew" that it would never be null, but we don't any more due to this effectively being exposed publicly.